README.md

```diff
@@ -6,7 +6,7 @@ Azure Functions extensions for Apache Kafka
 
 This repository contains Kafka binding extensions for the **Azure WebJobs SDK**. The extension status is experimental/under development. The communcation with Kafka is based on library **Confluent.Kafka**.
 
-**DISCLAIMER**: This library is under development and is experimental. Currently there is no guarantee regarding support or availability as a product.
+**DISCLAIMER**: This library is not currently supported on the Consumption Plan for Azure Functions. It is supported on Premium, where you need to manage scaling yourself, and also within KEDA where scaling will be handled for you based on the Kafka queue depth. 

```

Comment:
Suggested edits: 
", and also when using Azure Functions on Kubernetes (using KEDA) where scaling ... "

Reply:
@anirudhgarg should we add Premium and App service plan

Reply:
Premium is already there. We are really trying move people off the App Service plan so lets not add it.